src/main/java/core/basesyntax/BallColor.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum BallColor {
```

Comment:
```suggestion
public enum Color {
```
we can use this enum not only for balls